translations/en-us.yaml

```diff
@@ -0,0 +1 @@
+price_banner: The {product} costs {price, number, USD}
```

Comment:
looks like you have a couple things from ember-intl remaining that could be removed (including a few of the types files)